src/containers/stage.jsx

```diff
@@ -393,8 +393,8 @@ Stage.defaultProps = {
 const mapStateToProps = state => ({
     isColorPicking: state.colorPicker.active,
     isFullScreen: state.stageSize.isFullScreen,
-    // Do not use editor drag style in fullscreen mode.
-    useEditorDragStyle: !state.stageSize.isFullScreen
+    // Do not use editor drag style in fullscreen or player mode.
+    useEditorDragStyle: !(state.stageSize.isFullScreen || state.guiMode.mode === 'player')
```

Comment:
The `'player'` magic string should be imported (or use the `isPlayerMode` helper function I proposed).